.cache

```diff
@@ -0,0 +1 @@
+{"access_token": "BQDvtLPQnmpyn_YYLXU1Hk15WeVUOw2xPpoir9sZbbAet6iIG5Qs9uz3IOkVisCyuM1br6p4ozi2Bbh_-_P0iNx4L07TM27F0zaXbENcgROm2Xkg5ug", "token_type": "Bearer", "expires_in": 3600, "expires_at": 1664022315}
```

Comment:
please delete this/add .cache to git ignore

Reply:
ive added it . need help with this error though
'This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.'
thanks